app/models/project.rb

```diff
@@ -0,0 +1,61 @@
+class Project
+  include ActiveModel::Model
+
+  SAVE_URL = "https://academy-transfers-prototype-api.london.cloudapps.digital/projects".freeze
```

Comment:
Should this not be set globally via an environment variable and an initialiser or similar?

Reply:
I've raised a [new ticket](https://trello.com/c/b0UQqfon/616-change-how-api-urls-are-set-in-frontend-from-constants-to-environment-variable) to do this as it should be done in the other models too.